src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,45 @@
+import { Icon, IconKey } from "src/components/Icon";
+import { Css, Palette } from "src/Css";
+import { useTestIds } from "src/utils";
+import { IconButton } from "../IconButton";
+import { useToastContext } from "./ToastContext";
+
+export function Toast() {
+  const { setNotice, notice } = useToastContext();
+  const tid = useTestIds({}, "toast");
+  return (
+    <>
+      {notice && (
+        <div css={{ ...variantStyles[notice.type], ...Css.df.aic.w100.gap2.px2.py2.$ }} {...tid} role="alert">
```

Comment:
**request:** Per the figma mocks, it appears they gap should only be `8px`. You could also remove an additional "padding" definition by combining `py2.px2` to `p2`. 
```suggestion
        <div css={{ ...variantStyles[notice.type], ...Css.df.aic.w100.gap1.p2.$ }} {...tid} role="alert">
```

Reply:
Looking at this again, Figma says "basic" on it, but it doesn't appear to have the same definition in Figma that we have in Beam.... Probably fine for now. I'd just use `bshBasic` and maybe we update that globally.
<img width="1153" alt="Screen Shot 2022-12-01 at 12 08 58 PM" src="https://user-images.githubusercontent.com/1143861/205116010-fb2fa663-673a-4760-9dec-33b201251b7c.png">